lib/eofs/tests/test_solution.py

```diff
@@ -276,6 +276,34 @@ def _tomasked(self, value):
         return ma.MaskedArray(value, mask=np.isnan(value))
 
 
+class TestStandardDask(StandardSolutionTest):
+    weights = 'equal'
+
+    def _tomasked(self, value):
+        return ma.masked_invalid(value)
+
+    @classmethod
+    def modify_solution(cls):
+        import dask
```

Comment:
This means the test suite now requires dask or it will fail. This is not ideal, I don't want to make dask a required dependency at this stage. We should think about skipping this test when dask is not available. Not quite sure what the best approach would be right now.

Reply:
I would follow what xarray itself does here. Dask is an optional dependency

Something like:
```python
dask = pytest.importorskip('dask')
```
i.e. skip the tests where dask is not installed.

https://github.com/pydata/xarray/blob/master/xarray/tests/test_dask.py

Reply:
I've added `pytest.importorskip` to the Dask tests